packages/sdk/package.json

```diff
@@ -75,13 +74,14 @@
         "clean": "rimraf -- lib src/grpc-api"
     },
     "dependencies": {
-        "@concordium/rust-bindings": "2.0.0-rc.1",
+        "@concordium/rust-bindings": "2.0.0-rc.2",
         "@grpc/grpc-js": "^1.9.4",
         "@noble/ed25519": "^2.0.0",
         "@protobuf-ts/grpc-transport": "^2.9.1",
         "@protobuf-ts/grpcweb-transport": "^2.9.1",
         "@protobuf-ts/runtime-rpc": "^2.8.2",
         "@scure/bip39": "^1.2.1",
+        "@types/big.js": "^6.2.0",
```

Comment:
This is supposed to be a dev-dependency right?

Reply:
It's actually not, it needs to be there due to exporting `Big` types through the `CcdAmount` module.